File: Java/EstruturaSequencial/Exercicio12.java
// Tendo como dados de entrada a altura de uma pessoa, construa um algoritmo que calcule seu peso ideal, usando a seguinte fórmula: (72.7*altura) - 58
package EstruturaSequencial;
import java.util.Scanner;

public class Exercicio12 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Digite a altura da pessoa (em metros): ");
        double altura = scanner.nextDouble();

        double pesoIdeal = (72.7 * altura) - 58;

        System.out.printf("O peso ideal para a altura de %.2f metros é de %.2f kg.%n", altura, pesoIdeal);

        scanner.close();
    }
}

File: Python/EstruturaSequencial/exercicio_12.py
"""
Tendo como dados de entrada a altura de uma pessoa, construa um algoritmo que calcule seu peso ideal, usando a seguinte fórmula: (72.7*altura) - 58
"""

altura = float(input("Digite a altura da pessoa (em metros): "))
peso_ideal = (72.7 * altura) - 58

print(f"O peso ideal para a altura de {altura:.2f} metros é de {peso_ideal:.2f} kg.")
